_posts/2021/2021-01-11-read-ruby-version-in-your-gemfile.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Read the .ruby-version into your Gemfile"
+description: "A lovely little improvement to your life when upgrading Ruby"
+layout: article
+category: ruby
+image:
+  base: '2021/read-ruby-version-in-your-gemfile'
+  alt: 'Falling white dice'
+  credit:  Riho Kroll
+  source: "https://unsplash.com/photos/m4sGYaHYN5o"
+
+---
+
+Previously I recommended [using a loose version ruby version constraint in your Gemfile](/ruby/use-loose-ruby-versioning-in-your-gemfile/). This is still a valuable technique, but here’s another useful variation that's good for most of us, most of the time.
```

Comment:
*that _works well_ for most of us, most of the time ?

Reply:
```suggestion
Previously I recommended [using a loose version ruby version constraint in your Gemfile](/ruby/use-loose-ruby-versioning-in-your-gemfile/). This is still a valuable technique, but here’s another useful variation that's works well for most of us, most of the time.
```